frontend/__tests__/components/cluster-settings.spec.tsx

```diff
@@ -0,0 +1,139 @@
+  });
+  it('should render the Firehose Component with the props', () => {
+    expect(wrapper.find(Firehose).exists()).toBe(true);
+    expect(wrapper.find(Firehose).at(0).props().resources.length).toBe(2);
+    expect(wrapper.find(Firehose).at(0).props().resources[0].kind).toBe('config.openshift.io~v1~ClusterVersion');
+    expect(wrapper.find(Firehose).at(0).props().resources[1].kind).toBe('autoscaling.openshift.io~v1~ClusterAutoscaler');
+    expect(wrapper.find(Firehose).at(0).props().resources[0].name).toBe('version');
+    expect(wrapper.find(Firehose).at(0).props().resources[0].isList).toBe(false);
+    expect(wrapper.find(Firehose).at(0).props().resources[1].isList).toBe(true);
+  });
+  it('should render the HorizontalNav Component with the props', () => {
+    expect(wrapper.find(HorizontalNav).exists()).toBe(true);
+    expect(wrapper.find(HorizontalNav).at(0).props().pages.length).toBe(3);
+    expect(wrapper.find(HorizontalNav).at(0).props().hideDivider).toBe(true);
+    expect(wrapper.find(HorizontalNav).at(0).props().pages[0].name).toBe('Overview');
+    expect(wrapper.find(HorizontalNav).at(0).props().pages[1].name).toBe('Cluster Operators');
+    expect(wrapper.find(HorizontalNav).at(0).props().pages[2].name).toBe('Global Configuration');
+    expect(wrapper.find(HorizontalNav).at(0).props().pages[0].component).toEqual(ClusterVersionDetailsTable);
+    expect(wrapper.find(HorizontalNav).at(0).props().pages[1].component).toEqual(ClusterOperatorTabPage);
+    expect(wrapper.find(HorizontalNav).at(0).props().pages[2].component).toEqual(GlobalConfigPage);
+  });
+});
+
+describe('Cluster Version Details Table page', () => {
+  let wrapper : ShallowWrapper<any>;
+  let cv;
+
+  beforeEach(() => {
+    cv = clusterVersionProps;
+    wrapper = shallow(<ClusterVersionDetailsTable obj={cv} autoscalers={[]} />);
+  });
+
+  it('should render ClusterVersionDetailsTable component', () => {
+    expect(wrapper.exists()).toBe(true);
+  });
+  it('should render the child components of ClusterVersionDetailsTable component', () => {
+    expect(wrapper.find(CurrentChannel).exists()).toBe(true);
+    expect(wrapper.find(CurrentVersionHeader).exists()).toBe(true);
+    expect(wrapper.find(CurrentVersion).exists()).toBe(true);
+    expect(wrapper.find(UpdateLink).exists()).toBe(true);
+    expect(wrapper.find(UpdateStatus).exists()).toBe(true);
+    expect(wrapper.find(ResourceLink).exists()).toBe(true);
+    expect(wrapper.find(AddCircleOIcon).exists()).toBe(true);
+    expect(wrapper.find(Timestamp).exists()).toBe(true);
+  });
+  it('should render correct values of ClusterVersionDetailsTable component', () => {
+    expect(wrapper.find(CurrentChannel).at(0).props().cv.spec.channel).toEqual('stable-4.2');
+    expect(wrapper.find(CurrentVersion).at(0).props().cv.status.desired.version).toEqual('4.2.0-0.ci-2019-07-22-025130');
+    expect(wrapper.find('.co-break-all').at(0).text()).toEqual('342d8338-c08f-44ae-a82e-a032a4481fa9');
```

Comment:
Consider adding a`data-test-id` instead of using `.co-break-all` (here and below)